src/libraries/Microsoft.PowerFx.Interpreter/CustomFunction/ReflectionFunction.cs

```diff
@@ -157,6 +157,11 @@ private FunctionDescr Scan()
 
         private static FormulaType GetType(Type t)
         {
+            if (t == typeof(BlobValue))
+            {
+                return FormulaType.Blob;
+            }
+

```

Comment:
suspisious, What if it's Task<BlobValue>? Ie async custom function.

Reply:
it won't work :)